ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java

```diff
@@ -0,0 +1,262 @@
+package com.github.migi_1.ContextApp;
```

Comment:
This class looks like a god class. Maybe there is some way to move at least the initialization of element to a different class/ different classes